<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PathTab</class>
 <widget class="QWidget" name="PathTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Path Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_13">
        <item row="0" column="0">
         <widget class="QCheckBox" name="defaultPath_checkBox">
          <property name="text">
           <string>Use Default Relative Path</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="more_pb">
          <property name="text">
           <string>More</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_15">
        <item row="0" column="0">
         <widget class="QLabel" name="javaPath_label_2">
          <property name="text">
           <string>Java Path:</string>
          </property>
          <property name="buddy">
           <cstring>javaPath_le_2</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="javaPath_le_2"/>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="autoJava_pb_2">
          <property name="text">
           <string>Auto</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="javaPath_showPb_2">
          <property name="text">
           <string>Show...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_12">
        <item row="8" column="2">
         <widget class="QPushButton" name="indexesPath_showPb">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Show...</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="assetsPath_le">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="corePath_le">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="corePath_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Core Path:</string>
          </property>
          <property name="buddy">
           <cstring>corePath_le</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="corePath_showPb">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Show...</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="versionsPath_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Version Folder Path:</string>
          </property>
          <property name="buddy">
           <cstring>versionsPath_le</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="libsPath_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Libraries Path:</string>
          </property>
          <property name="buddy">
           <cstring>libsPath_le</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="libsPath_showPb">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Show...</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="versionsPath_le">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="nativesPath_le">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="assetsPath_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Assets Path:</string>
          </property>
          <property name="buddy">
           <cstring>assetsPath_le</cstring>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="nativesPath_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Natives Extract Path:</string>
          </property>
          <property name="buddy">
           <cstring>nativesPath_le</cstring>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QPushButton" name="nativesPath_showPb">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Show...</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="libsPath_le">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="versionsPath_showPb">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Show...</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="indexesPath_le">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="objectsPath_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Asset Objects Path:</string>
          </property>
          <property name="buddy">
           <cstring>objectsPath_le</cstring>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLineEdit" name="objectsPath_le">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QPushButton" name="objectsPath_showPb">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Show...</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QPushButton" name="assetsPath_showPb">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Show...</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="indexesPath_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Asset Indexes Path:</string>
          </property>
          <property name="buddy">
           <cstring>indexesPath_le</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="2" rowspan="3">
         <widget class="QPushButton" name="defaultGamePath_showPb">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Show...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="3">
         <widget class="QLabel" name="defaultGamePath_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Default Game Path:</string>
          </property>
          <property name="buddy">
           <cstring>corePath_le</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1" rowspan="3">
         <widget class="QLineEdit" name="defaultGamePath_le">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="unusedModsPath_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Unused Mods Path:</string>
          </property>
          <property name="buddy">
           <cstring>unusedModsPath_le</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="loggerPath_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Logger Path:</string>
          </property>
          <property name="buddy">
           <cstring>loggerPath_le</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="savesBackupPath_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Saves Backup Path:</string>
          </property>
          <property name="buddy">
           <cstring>savesBackupPath_le</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="loggerPath_le">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="unusedModsPath_le">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="unusedModsPath_showPb">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Show...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="savesBackupPath_showPb">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Show...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="savesBackupPath_le">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="loggerPath_showPb">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Show...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="customPathHelper_tb">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Core Path:&amp;lt;core&amp;gt;&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Assets Path:&amp;lt;assets&amp;gt;&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Launcher Path:&amp;lt;launcher&amp;gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>17</width>
          <height>68</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
